pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -529,7 +530,15 @@ in buildPythonPackage {
     keras
     portpicker
     tblib
+    load_libstdcxx
   ];
+
+  pythonImportsCheck = [
+    "tensorflow"
+    "tensorflow.compat.v1"
+    "load_libstdcxx"
```

Comment:
is the order meaningful here?

Reply:
Yes, `load_libstdcxx` should always come last. Needs a comment